tests/nextflow.config

```diff
@@ -0,0 +1,29 @@
+/*
+========================================================================================
+    Nextflow config file for running tests
+========================================================================================
+*/
+
+params {
+    outdir = 'results'
+}
+
+// Impose sensible resource limits for testing
+process {
+    withName: '.*' {
+        cpus   = 2
+        memory = 3.GB
+        time   = 2.h
+    }
+}
+
+// Impose same minimum Nextflow version as the pipeline for testing
+manifest {
+    nextflowVersion = '!>=23.04.0'
```

Comment:
You used `>=24.04.1` on the pipeline `nextflow.config`

Reply:
ups, updated here too

Reply:
where was this updated?

Reply:
here --> https://github.com/nf-core/multiplesequencealign/pull/161